projects/addon-editor/utils/get-mark-range.ts

```diff
@@ -0,0 +1,82 @@
+import type {Mark as ProseMirrorMark, MarkType, ResolvedPos} from 'prosemirror-model';
+
+export type Range = {
```

Comment:
Can it be an Array tuple?

Reply:
No, but we can import type from tiptap core :+1: